test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,12 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const c = net.createConnection(0, '***');
```

Comment:
Specifically, sending to a valid port that's not listening will end up getting `ECONNREFUSED` but sending port 0 will end up getting `EADDRNOTAVAIL`.